web/components/landing-page-panel.tsx

```diff
@@ -1,64 +1,451 @@
+          {pageNumber === 0 && <LandingPage0 isMobile={isMobile} />}
+          {pageNumber === 1 && <LandingPage1 isMobile={isMobile} />}
+          {pageNumber === 2 && <LandingPage2 isMobile={isMobile} />}
+          {!isMobile && (
+            <div className="absolute right-0 bottom-0 z-20 h-full">
+              <SquiggleVerticalIcon className={clsx('text-indigo-200')} />
+            </div>
+          )}
+          {isMobile && (
+            <div className="absolute right-0 -bottom-0.5 z-20 w-full items-center">
+              <SquiggleHorizontalIcon className={clsx('text-indigo-200')} />
+            </div>
+          )}
+          <div
+            className={clsx(
+              'absolute',
+              isMobile ? 'right-0 h-full w-8' : 'bottom-0 h-6 w-full'
+            )}
+          >
+            <div
+              className={clsx(
+                'z-50 gap-3',
+                isMobile ? 'mt-40 ml-3 flex flex-col' : 'ml-40 flex flex-row'
+              )}
+            >
+              <PaginationCircle
+                currentPageNumber={pageNumber}
+                pageNumber={0}
+                onClick={() => {
+                  setPageNumber(0)
+                }}
+              />
+              <PaginationCircle
+                currentPageNumber={pageNumber}
+                pageNumber={1}
+                onClick={() => setPageNumber(1)}
+              />
+              <PaginationCircle
+                currentPageNumber={pageNumber}
+                pageNumber={2}
+                onClick={() => setPageNumber(2)}
+              />
             </div>
-          </h1>
-          <Spacer h={6} />
-          <div className="mb-4 px-2 ">
-            Create a play-money prediction market on any topic you care about.
-            Trade with your friends to forecast the future.
-            <br />
           </div>
         </div>
-        <Spacer h={6} />
-        <button
-          className="self-center rounded-md border-none bg-gradient-to-r from-indigo-500 to-blue-500 py-4 px-6 text-lg font-semibold normal-case text-white hover:from-indigo-600 hover:to-blue-600"
-          onClick={withTracking(firebaseLogin, 'landing page button click')}
+        <div
+          className={clsx(
+            'z-30 w-full bg-indigo-200 sm:w-2/5',
+            isMobile ? 'h-1/5 rounded-b-xl' : `${desktop_height} rounded-r-xl`
+          )}
         >
-          Get started
-        </button>{' '}
+          <LandingPageManifoldMarketsLogo isMobile={isMobile} />
+          <div
+            className="absolute bottom-16 right-8 z-30 md:right-12"
+            onMouseEnter={() => setIsButtonHovered(true)}
+            onMouseLeave={() => setIsButtonHovered(false)}
+          >
+            <Button
+              className={clsx(
+                'absolute z-10 transition-all ease-in-out focus:bottom-0.5 focus:-left-0.5',
+                isButtonHover ? 'bottom-2 -left-2' : 'bottom-1.5 -left-1.5'
```

Comment:
I believe you can accomplish this hover effect with css. This is desirable because javascript listeners are slower and more laggy.

https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state

In short, mark the parent div with the `group` class, then add the `group-hover:` prefix to classes that should apply on hover.

Reply:
Oh this is really helpful!